### DISCUSSION TITLE
Reorder task groups by drag and drop

### DISCUSSION THREAD

**Discussion by OlegLoewen** [lawand-dot-io/taskbar#77] [Feature requests]
Currently reordering is disabled, if "Group tasks by app" is enabled. However, reordering the tasks groups per drag and drop would still be a great feature.

**Comment by lawand-dot-io** [comment]
Thanks for the suggestion. I had this in mind while implementing task grouping. hopefully I'll implement this at some point 👍 as always the most upvoted requests are prioritized. by the way a quick solution would be to move all app windows to another Space then back. this makes the app group moved to the end of the list. cheers

**Comment by lawand-dot-io** [comment]
Duplicate: https://github.com/lawand-dot-io/taskbar/discussions/124

**Comment by chakkarss** [comment]
This would be a very good feature and increase productivity a lot.
